app/controllers/application_controller/explorer.rb

```diff
@@ -221,7 +221,7 @@ def x_build_dynatree(options)
     # :full_ids               # stack parent id on top of each node id
 
     roots = x_get_tree_objects(options.merge(
-                                 :userid => session[:userid], # Userid for RBAC filtering
+                                 :user   => current_user,     # Userid for RBAC filtering
```

Comment:
@kbrock i am working on cleaning up https://github.com/ManageIQ/manageiq/blob/master/app/controllers/application_controller/explorer.rb to remove any of the tree building code that we no longer need.